tests/quadrature/quadrature_test.C

```diff
@@ -511,6 +515,10 @@ public:
   template <QuadratureType qtype, Order order, unsigned int exactorder>
   void test3DWeights ()
   {
+    // Our Gauss-Lobatto rules are only computed to double precision
```

Comment:
Wait, why? They should have like 32 *accurate* decimal digits.

Reply:
We've got a bunch of 32 decimal place literals, but they're all suffixed with L, which truncates to 20 digits when it interprets them as long double.  If we really want all the precision we can get, we'll need a new suffix that we can shim to the boost suffix when we have it or to L when we don't.

Since those unit tests use ```TOLERANCE*sqrt(TOLERANCE)``` and I only set ```TOLERANCE = 1e-11```, 20 digits was good enough for most cases.  But we don't have 20 digits everywhere.

For Gaussian quadrature on Tets we only have 15 digits at second order:
https://github.com/libMesh/libmesh/blob/17be2c62003504c8555d7af2400fc7fd2d5e5924/src/quadrature/quadrature_gauss_3D.C#L85
and likewise on Triangles at seventh order:
https://github.com/libMesh/libmesh/blob/17be2c62003504c8555d7af2400fc7fd2d5e5924/src/quadrature/quadrature_gauss_2D.C#L338

And... what the hell.  I could swear that I saw a similar case in the Gauss-Lobatto rules, and that that prompted me to rule out those specific unit tests, but now I can't find it.  Let me reenable them and see what happens.

Reply:
I do get failures again.  In both 2D and 3D, for both ```SIXTH``` and ```SEVENTH``` order.

OH HELL.  There's a ```0.1``` in gauss_lobatto_quadrature_1D.C I need to change to ```1/Real(10)```, isn't there?

Reply:
And that fixes the tests for me.  I'll upload in a minute.

Reply:
Thanks for catching this!!

Reply:
> Thanks for catching this!!

Sure, but I think you are the one who caught the but that I apparently left as an easter egg in 3998bf26f.

Reply:
> If we really want all the precision we can get, we'll need a new suffix that we can shim to the boost suffix when we have it or to L when we don't.

I scanned the boost.multiprecision introduction and "Literal types support" docs. It looks like only the only supported user-defined literals are currently `_cppi`, `_cppui`, etc. which are all for integral types. 

According to [this](https://stackoverflow.com/questions/33043809/what-is-the-precision-of-cpp-dec-float-50) it looks like you can pass a string to e.g. the `boost::multiprecision::cpp_dec_float_50` constructor. Maybe it's possible to come up with a simple `to_real()` macro that would do the right thing (stringification, concatenation, etc.) in all cases from the same input?

```C
#ifdef LIBMESH_DEFAULT_QUAD_PRECISION
#define to_real(x) boost::multiprecision::cpp_dec_float_50("#x")
#else
#define to_real(x) long double(x##L)
#endif
```
```
_weights[0] = to_real(6.6666666666666666666666666666667e-02);
```

Reply:
Just doing stringification in a macro seems clever.  I'd just been *assuming* that we needed to use user-defined literals, but I guess as long as the cpp_dec_float constructor and the conversion to float128 are both constexpr then it'll work out just fine in the end.